controllers/api/v1/keyword_test.go

```diff
@@ -0,0 +1,168 @@
+package apiv1
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	_ "google-scraper/initializers"
+	. "google-scraper/tests"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gstruct"
+	"github.com/tidwall/gjson"
+)
+
+type KeywordListResponse struct {
+	Id              int64
+	Name            string
+	SearchCompleted bool
+	CreatedAt       string
+	Meta            gjson.Result
+	Links           gjson.Result
+}
+
+var _ = Describe("KeywordController", func() {
+	Describe("GET /api/v1/keywords", func() {
+		Context("Given valid credentials", func() {
```

Comment:
Should this be a small `g` to make it consistent with another context? Also please check another context too.

> given valid credentials